BuildCliComponents.cmd

```diff
@@ -40,13 +43,13 @@ if NOT exist "%DotNet%" (
 echo Building CLI-based components
 pushd %~dp0src\cli\Microsoft.DotNet.xunit.performance.runner.cli
 call %DotNet% restore
-call %DotNet% build -c Release
-call %DotNet% pack -c Release -o %OutputDirectory%
+call %DotNet% build -c %BuildConfiguration%
+call %DotNet% pack -c %BuildConfiguration% -o %OutputDirectory%
 popd
 pushd %~dp0src\cli\Microsoft.DotNet.xunit.performance.analysis.cli
 call %DotNet% restore
-call %DotNet% build -c Release
-call %DotNet% pack -c Release -o %OutputDirectory%
+call %DotNet% build -c %BuildConfiguration%
+call %DotNet% pack -c %BuildConfiguration% -o %OutputDirectory%
```

Comment:
Not sure if we want to pack a Debug version, maybe we can pack only if it is a Release build